cmd/storage.go

```diff
@@ -52,16 +52,22 @@ var storageCreateCmd = &cobra.Command{
 }
 
 var storageUnmountCmd = &cobra.Command{
-	Use:   "unmount [storage name]",
-	Short: "Unmount storage from the current component",
-	Long: `Unmount storage from the current component.
+	Use:   "unmount [PATH | STORAGE_NAME]",
```

Comment:
this should be lowercase, other sources (such as `create.go`) use lowercase.

`unmount <path | storage_name>` maybe better?

Reply:
@cdrage see https://github.com/redhat-developer/odo/issues/516

there are two options, either uppercase, or lowercase in `<>`

Reply:
@mik-dass it can't be in `[]` as it means that it is optional. Please check rules in  #516

Reply:
@kadel So between <>?

Reply:
As described in docopts docs (link in #516) 

either just capital letters 
```
unmount PATH | STORAGE_NAME
```

or in `<>`
```
unmount <path> | <storage_name>
```

I would use the capital letters, it  seems to be more widly used (heroku and could foundry).
We should pick one and stick with it across all commands in OpenShift-Do.

Reply:
@kadel OK I will changed to the Upper case approach